tools/erc-repository-indexer/erc-contract-indexer/src/services/contractScanner.ts

```diff
@@ -35,9 +35,8 @@ export class ContractScannerService {
    */
   private readonly mirrorNodeBaseUrl: string;
 
-  constructor() {
-    const network = process.env.HEDERA_NETWORK || 'previewnet';
-    this.mirrorNodeBaseUrl = `https:${network}.mirrornode.hedera.com`;
+  constructor(hederaNetwork: string) {
+    this.mirrorNodeBaseUrl = `https:${hederaNetwork}.mirrornode.hedera.com`;
```

Comment:
I would make mirror node url a config.
Mainnet for example is mainnet-public and not mainnet.
Plus if you do that you allow for custom mirror node urls

Reply:
Gotcha I agree and will restore the MIRROR_NODE_URL config.

> Mainnet for example is mainnet-public and not mainnet.

Hmm I believe the correct address is still mainnet.mirrornode.hedera.com as they are documented on the  doc page at https://docs.hedera.com/hedera/sdks-and-apis/rest-api and the API documentation at https://mainnet.mirrornode.hedera.com/api/v1/docs/.

Reply:
Restored MIRROR_NODE_URL config.